src/Intro.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import key from './api/api.js';
+
+function Intro () {
+
+  let city = '';
```

Comment:
if you're using this to manage state, use the useState hook

https://reactjs.org/docs/hooks-state.html

Reply:
I decided to refactor to use the useState hook. To clarify, this is to take advantage of state and state management in React?

Reply:
Do you understand what the use of state is in React?

Reply:
Decided to look and read into state again (https://reactjs.org/docs/faq-state.html). State holds information and re-renders the page upon change. The second part was what I overlooked, thanks!

Reply:
yep, essentially, you don't need to use state if you need to hold information that doesn't re-render the page - but in this case, you would want your weather display to change whenever city changes, so perfect use case for state